fastlane/lib/fastlane/runner.rb

```diff
@@ -258,12 +258,11 @@ def execute_flow_block(block, current_platform, lane, parameters)
 
     def verify_supported_os(name, class_ref)
       if class_ref.respond_to?(:is_supported?)
-        if Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
```

Comment:
if by other changes you mean 

* those in this file, then it's just reordering the code to reduce code duplication

* if it is about the other changes, they allow the user to add a new (unsupported) platform in her Fastfile without having the platform to crash. See the overall PR description to see how we use it today at my work place.